Library/Formula/chadwick.rb

```diff
@@ -0,0 +1,24 @@
+class Chadwick < Formula
+  desc "Chadwick tools for parsing Retrosheet MLB play-by-play files."
+  homepage "http://chadwick.sourceforge.net/doc/index.html"
+  url "https://downloads.sourceforge.net/project/chadwick/chadwick-0.6/chadwick-0.6.4/chadwick-0.6.4.tar.gz"
+  sha256 "2fcc44b4110c3aebf8b9f3daaccf22ccaba3760d3d878e65d38b86127b913559"
+
+  def install
+    system "./configure", "--prefix=#{prefix}", "--disable-debug", "--disable-dependency-tracking"
+    system "make"
+    system "make", "install"
+  end
+
+  resource "event_files" do
```

Comment:
This is a nit, but `resource`s should go above `def install`. They'd usually go under the `depends_on` block but obviously there is no `depends_on` block here :wink: